components/Button/Button.tsx

```diff
@@ -39,8 +39,42 @@ export const SwishButton = ({
                 <div className={styles.swishBackground} />
             </div>
             <div className={styles.text}>{children}</div>
-            <ChevronIcon className={styles.chevron} />
+            <ChevronIcon className={styles.endingChevron} />
             {jobs > 0 && <div className={styles.jobsBadge}>{jobs}</div>}
         </a>
     </Link>
 );
+
+type CarouselArrowButtonProps = {
+    direction: 'LEFT' | 'RIGHT';
+    carouselButtonRef: React.RefObject<HTMLButtonElement>;
+    className?: string;
+};
+
+export const CarouselArrowButton = ({
+    direction,
+    carouselButtonRef,
+    className,
+}: CarouselArrowButtonProps) => {
+    const chevron =
+        direction === 'LEFT' ? (
+            <ChevronIcon
+                className={`${styles.centerChevron} ${styles.centerChevronFacingLeft}`}
+            />
+        ) : (
+            <ChevronIcon className={styles.centerChevron} />
+        );
+
+    return (
+        <button
+            type="button"
+            ref={carouselButtonRef}
+            className={[styles.button, styles.chevronButton, className].join(
+                ' ',
+            )}
+            aria-hidden
```

Comment:
**Question**: Is this okay for an interactive element?

Reply:
The carousel buttons are hidden as screenreader users will read the carousel contents directly